p4-fusion/commands/change_list.cc

```diff
@@ -6,27 +6,27 @@
  */
 #include "change_list.h"
 
+#include <utility>
+
 #include "p4_api.h"
 #include "describe_result.h"
 #include "filelog_result.h"
 #include "print_result.h"
 #include "utils/std_helpers.h"
 #include "minitrace.h"
 
-#include "thread_pool.h"
-
-ChangeList::ChangeList(const std::string& clNumber, const std::string& clDescription, const std::string& userID, const int64_t& clTimestamp)
-    : number(clNumber)
-    , user(userID)
-    , description(clDescription)
+ChangeList::ChangeList(std::string  clNumber, std::string  clDescription, std::string  userID, const int64_t& clTimestamp)
+    : number(std::move(clNumber))
+    , user(std::move(userID))
+    , description(std::move(clDescription))
     , timestamp(clTimestamp)
     , changedFileGroups(ChangedFileGroups::Empty())
 {
 }
 
 void ChangeList::PrepareDownload(P4API* p4, const BranchSet& branchSet)
 {
-	MTR_SCOPE("ChangeList", __func__);
+	MTR_SCOPE("ChangeList", __func__)
```

Comment:
For consistency sake, I think it's better to leave the semicolons in, as incorrect changes to the macro definition can lead to very confusing error messages if you omit the `;`. The compiler can emit a more useful error message if parsing is terminated earlier with a `;`.